### ISSUE TITLE
Geometry attached to Ground body is now lost

### ISSUE THREAD

**Issue by aymanhab** [opensim-org/opensim-core#419]
Deserialization does not handle moving old Geometry that used to live in Ground body to the one and only Ground frame in the model. This causes testVisualization to fail and will cause almost all our old models not to display correctly.

**Comment by aseth1** [comment]
I am glad a testVisualization is being added. It is likely that some Geometry on what used to be the "ground" Body is not being added to the Ground frame. I had modified `Model::extendFinalizeFromProperties()` [here ](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Simulation/Model/Model.cpp#L543) in Model.cpp to check for a "ground" Body and to add its VisibleObject and WrapObjectSet to the Ground. I see I made an error in there, in that the VisibleObject does not get added to the Ground eventhough it was fished out of the Body. I know the case of Arm26 tests were failing without having the wrap objects added to the Ground (since the Torso is Ground) so I (incorrectly) assumed I got the VisibleObject when I added the  wrap objects. Sorry about that!
